src/torchmetrics/classification/accuracy.py

```diff
@@ -450,18 +451,31 @@ def __new__(
         validate_args: bool = True,
         **kwargs: Any,
     ) -> Metric:
+
         kwargs.update(
             {"multidim_average": multidim_average, "ignore_index": ignore_index, "validate_args": validate_args}
         )
+
+        if task not in ["binary", "multiclass", "multilabel"]:
```

Comment:
let's wait for #1479 is merged and use this Enum

Reply:
I see that 1479 is merged. Is there anything else I need to do?

Reply:
Can you please use these enums?

Reply:
implemented in https://github.com/Lightning-AI/metrics/pull/1448/commits/a1e118fd894a928d0ee60d4aca38db8a8477c9d1